docs/calico-with-docker/docker-network-plugin/README.md

```diff
@@ -281,8 +291,11 @@ For more details, read
 [Accessing Calico policy with Calico as a network plugin](AdvancedPolicy.md).
 
 ## Make a container reachable from the Host-Interface (Internet)
-
-You cannot simply use `-p`on `docker run` to expose ports. We have a working example on how to [expose a container port to the internet](../../ExposePortsToInternet.md)
+If you're interested in using port-forwarding, we have a working example on how
+to [expose a container port to the internet](../../ExposePortsToInternet.md)
+when using Calico.  Note that calling `docker run` with the `-p` flag will not
```

Comment:
This second sentence adds a lot of FUD.  I'd remove it.  Also I'm not really sure "secured by Calico" really means what you think.